src/index.ts

```diff
@@ -108,6 +108,23 @@ export const createFakeEggs = ({
     word: createWordGenerator(chance),
     zip: createZipGenerator(chance),
   };
+
+  type FakeEggsCustomMethods = typeof customMethods;
+  interface FakeEggs
```

Comment:
A little awkward to have the type definition here, but it does achieve the desired result of a return type that overrides chance's method types for each of our custom methods.